Plugins docs (#21826)
grafana/grafana: docs/sources/plugins/developing/apps.md

# Grafana Apps

> App plugins are a grafana plugin that can bundle data source and panel plugins within one package. It also enables the plugin author to create custom pages within grafana. The custom pages enable the plugin author to include things like documentation, sign up forms, or controlling other services using HTTP requests.

Reviewer: ```suggestion
App plugins are a Grafana plugin that can bundle data source and panel plugins within one package. They also enable the plugin author to create custom pages within Grafana. The custom pages enable the plugin author to include things like documentation, sign-up forms, or to control other services with HTTP requests.
```

Reviewer: If our reader is the plugin author, then we should change "plugin author" to "you" throughout.

Author: These are more so just my suggestions. First to admit this is new territory lol
